src/main/index.js

```diff
@@ -171,7 +171,7 @@ function runApp() {
     }
   }
 
-  async function createWindow(replaceMainWindow = true) {
+  async function createWindow({ replaceMainWindow = true, windowStartupUrl = null, showWindowNow = false } = { replaceMainWindow: true, windowStartupUrl: null, showWindowNow: false }) {
```

Comment:
You can avoid code repetition by doing this
```suggestion
  async function createWindow({ replaceMainWindow = true, windowStartupUrl = null, showWindowNow = false } = { }) {
```